### PR TITLE
Account page now renders skills panel

### PR DESCRIPTION
**Pull request by the-code-raccoon**
Also includes working shitty looking sorting
image

### CODE REVIEW

**Comment by Gahmbit** [comment]
bro cooked fr

File: frontend/src/assets/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import agilityIcon from "./agility-icon.png";
+import archaeologyIcon from "./archaeology-icon.png";
+import attackIcon from "./attack-icon.png";
+import constitutionIcon from "./constitution-icon.png";
+import constructionIcon from "./construction-icon.png";
```
Comment (Gahmbit) [inline frontend/src/assets/index.tsx:5]: duplicate

File: frontend/src/assets/index.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import craftingIcon from "./crafting-icon.png";
+import defenceIcon from "./defence-icon.png";
+import divinationIcon from "./divination-icon.png";
+import dungeoneeringIcon from "./dungeoneering-icon.png";
+import farmingIcon from "./farming-icon.png";
+import firemakingIcon from "./firemaking-icon.png";
+import fishingIcon from "./fishing-icon.png";
+import fletchingIcon from "./fletching-icon.png";
+import herbloreIcon from "./herblore-icon.png";
+import highIcore from "./high-score-icon.png";
```
Comment (Gahmbit) [inline frontend/src/assets/index.tsx:16]: rename to highIcon

File: frontend/src/pages/LandingPage.tsx [hunk 3]
```diff
@@ -28,17 +28,17 @@ const LandingPage = () => {
...
-                            ref={searchBar}
-                            id="rsn"
-                            name="rsn"
-                            placeholder="Enter your RuneScape Username"
-                        />
-                        <button type="button" onClick={loadAccountPage}>
-                            Search
-                        </button>
-                    </div>
+                    <form onSubmit={loadAccountPage}>
```
Comment (Gahmbit) [inline frontend/src/pages/LandingPage.tsx:31]: why a form again?

**Review by Gahmbit — COMMENTED** [review]
cooked 10/10

File: frontend/src/pages/LandingPage.tsx (hunk 3 above)
Comment (the-code-raccoon) [reply frontend/src/pages/LandingPage.tsx:31]: so after user puts something in the search bag you can press enter to load the account page

**Pull request merged**